spec/poros/food_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Food do
+
+  it "exists and has attributes" do
+    attributes = {
+      description: 'sweet potatoes',
+      gtinUpc: '492111402857',
+      brandOwner: 'ARCHER FARMS',
+      ingredients: 'SWEET POTATOES'
+    }
+
+    food_info = Food.new(attributes)
+
+    expect(food_info).to be_an_instance_of(Food)
+    expect(food_info.description).to eq('sweet potatoes')
```

Comment:
Nice assertions here -- Here it's totally fine to expect the exact values because we are stubbing the hash we pass in. So if you use fixtures, or use just webmock in other tests, exact value testing is fine for external API data. But, if you use VCR or don't have a mock set up, then that data is likely to change often and might break your tests if you expect exact values. 

But again, for the purposes of an assessment, it's very reasonable to test in this way as there is limited time.